README.md

```diff
@@ -27,6 +27,34 @@ function mystery(n) {
 }
 ```
 
+Recurrence relation:
+
+T(n)=1 if n ≤ 1 or 3T(n/3) + $n^5$  if n > 1
+
+Solve by substitution:
+
+
+$T(n)= 3T(n/3) + n^5$
+
+$3(3T(n/9) + n^5) + n^5$
+      
+$9T(n/9) + 2n^5$
```

Comment:
This step is incorrect -- you need to multiply everything in the parentheses by 3.